src/components/app.tsx

```diff
@@ -471,7 +480,8 @@ export class App extends React.PureComponent<IProps, IState> {
     } else if (page >= 0 && (activity && page <= activity.pages.length)) {
       const navigateAway = () => {
         this.setState({ currentPage: page, incompleteQuestions: [] });
-        setDocumentTitle({activity, pageNumber: page});
+        setDocumentTitle(activity, page);
```

Comment:
This line should be changed back to `setDocumentTitle({activity, pageNumber: page});`.